### ISSUE TITLE
Using GitBook instead of the software-carpentery clone?

### ISSUE THREAD

**Issue by RaoOfPhysics** [lhcb/first-analysis-steps#43]
You have have already seen [GitBook](https://www.gitbook.com/) used at CERN:
- https://cern.ch/clouddocs/
- https://cern.ch/configdocs/

It's probably a better tool for this document, as you can also allow the reader to change the font and background colours for better readability, but you can also create a built-in glossary to refer to.

The configdocs file I pointed to also has instructions for installing GitBook at CERN: https://configdocs.web.cern.ch/configdocs/gbook/README.html

**Comment by betatim** [comment]
Hi, thanks for the pointer. Could you elaborate (or even better build a prototype) on what the advantages would be? The template works very well for us so far and all the SWC lessons it seems.

Right now we are busy preparing the actual material with a workshop coming up. Maybe in the future we can consider switching.

**Comment by ibab** [comment]
Gitbook looks like a very nice choice. I can test it after the workshop next week.

**Comment by RaoOfPhysics** [comment]
@betatim: I guess it's just more user-friendly, because one can change the colours and fonts, as I mentioned. Check out the built-in glossary on the CERN instances of GitBook: saves some trouble when you need to refer to a specific term repeatedly.

I can try porting your work to GitBook as an example, but I'm not an LHCb person and don't have the free time to do this just yet. :)

**Comment by alexpearce** [comment]
I'm trialling GitBook (and the possibility of merging first and second-analysis-steps) in [lhcb/starterkit-lessons](https://github.com/lhcb/starterkit-lessons).

**Comment by henryiii** [comment]
It looks pretty good. I could help merge DevelopKit in too if this works out.

**Comment by alexpearce** [comment]
Maybe the discussion is better had on that repo, but generally I think it's OK. I'm not sure about having all the lessons visible in the sidebar, I'm not sure if things start to look too cluttered. But it's better for visibility than having the lessons on separate websites, I suppose.

**Comment by RaoOfPhysics** [comment]
@alexpearce:

> I'm not sure about having all the lessons visible in the sidebar, I'm not sure if things start to look too cluttered.

A reader can collapse them so they're out of the way, using the "hamburger menu".

**Comment by alexpearce** [comment]
@RaoOfPhysics True, but I more meant about having a very long list of lessons to scroll through. @chrisburr has a nice solution in lhcb/starterkit-lessons#1.
